enclave_wrangler/objects_api.py

```diff
@@ -570,7 +589,20 @@ def cli():
     #     run(**kwargs_dict)
 
 
-def get_concept_set_version_expression_items(version_id: Union[str, int], return_detail=['id', 'full'][0]):
+# todo: @Siggie: is this redundant with get_concept_set_version_expression_items() below?
```

Comment:
`get_expression_items()`
`# todo: @Siggie: is this redundant with get_concept_set_version_expression_items() below?`